client/webserver/websocket.go

```diff
@@ -144,9 +145,12 @@ func (s *WebServer) handleMessage(conn *wsClient, msg *msgjson.Message) *msgjson
 	return msgjson.NewError(msgjson.UnknownMessageType, "web server only handles requests")
 }
 
+// All request handlers must be defined with this signature.
+type wsHandler func(*WebServer, *wsClient, *msgjson.Message) *msgjson.Error
```

Comment:
This shuts up linter warnings about `wsUnmarket` and `wsAckNotes` always returning `nil` even though they had to in order to fit in the map.  I guess this makes is that much clear to the linter.